components/org.wso2.transport.remote-file-system/src/main/java/org/wso2/transport/remotefilesystem/server/RemoteFileSystemConsumer.java

```diff
@@ -242,8 +242,8 @@ private void directoryHandler(FileObject[] children)
      * @param file A single file to be processed
      */
     private void fileHandler(FileObject file) throws FileSystemException {
-        String path = file.getName().getPath();
-        if (processed.contains(file.getName().getPath())) {
+        String path = file.getName().getURI();
```

Comment:
this is redundant

Reply:
Fixed with https://github.com/wso2/transport-file/pull/27/commits/3a24c99f9b48594ab0c2fbc48586389021a9e168